lib/test_track_rails_client/rspec_helpers.rb

```diff
@@ -2,4 +2,8 @@
 
 RSpec.configure do |config|
   config.include TestTrackRailsClient::AssignmentHelper
+
+  config.before(:each) do
+    TestTrack::FakeServer.reset!(RSpec.configuration.seed)
```

Comment:
I suppose we could continue to stub the remote split registry here: https://github.com/Betterment/test_track_rails_client/pull/24/files#diff-eca6a7243d1a365a9354cf12853389bcL17.

But then the ruby and js clients would see different split registries.